Makefile

```diff
@@ -401,7 +401,7 @@ else
 	strip --strip-unneeded target/$(HOST_TARGET)/release/wasmer-headless
 endif
 
-WAPM_VERSION = v0.5.3
+WAPM_VERSION = v0.5.9
```

Comment:
We need to remove this

Reply:
Done in https://github.com/wasmerio/wasmer/pull/3432/commits/964b897133b2a311cfd1d6aafa7a916e99df428b